test-cases/scylla-operator/functional.yaml

```diff
@@ -4,7 +4,7 @@ user_prefix: 'functional'
 n_db_nodes: 4
 k8s_n_scylla_pods_per_cluster: 3
 
-n_loaders: 0
+n_loaders: 1
```

Comment:
Maybe also make it be dynamic loader?

Reply:
right now it's using docker backend, how would I use dynamic ? i.e. it's not just flipping a configuration.

Reply:
Ok, looked at the code and indeed it is set to be using docker. I think it is not what should be done.
So, let it be as is for the moment. 

But later we should switch to K8S-based loader workloads.